packages/fxa-graphql-api/src/lib/db/index.ts

```diff
@@ -28,8 +25,71 @@ export async function dbHealthCheck(): Promise<HealthExtras> {
   };
 }
 
-export function setupDatabase(opts: DatabaseConfig): Knex {
-  const knex = Knex({ connection: opts, client: 'mysql' });
-  Model.knex(knex);
+/**
+ * Setup a connection after knex establishes it, before its used.
+ *
+ * @param conn rawDriverConnection from knex
+ * @param done callback to indicate connection is ok
+ */
+async function setupConnection(conn: any, done: any) {
+  const query = promisify(conn.query).bind(conn);
+  try {
+    // Always communicate timestamps in UTC.
+    await query("SET time_zone = '+00:00'");
+    // Always use full 4-byte UTF-8 for communicating unicode.
+    await query('SET NAMES utf8mb4 COLLATE utf8mb4_bin;');
+    // Always have certain modes active. The complexity here is to
+    // preserve any extra modes active by default on the server.
+    // We also try to preserve the order of the existing mode flags,
+    // just in case the order has some obscure effect we don't know about.
```

Comment:
This is actually copy/paste from profile/auth/auth-db mysql setup code that exists everywhere we talk to a MySQL db in FxA.

Reply:
(FWIW the most important one of these is "force the db to run in strict mode", because running without strict mode can (and has!) lead to security problems)